dom.go

```diff
@@ -37,6 +37,14 @@ type Component interface {
 	Context() *Core
 }
 
+// Unmounter is an optional interface that a Component can implement in order
+// to receive component unmount events.
+type Unmounter interface {
+	// Unmount is called after the component has been unmounted, in a separate
+	// goroutine, after the DOM element has been removed.
```

Comment:
Indeed I did! 😨 Thanks for pointing this out! 😃 

Fixed in becba2706b0445067f6cc51d9975846d946c8f6c